six/two/two.cpp

```diff
@@ -377,12 +377,17 @@ PyObject *Two::_findSubclassOf(PyObject *base, PyObject *module)
     PyObject *klass = NULL;
     for (int i = 0; i < PyList_GET_SIZE(dir); i++) {
         // get symbol name
-        char *symbol_name;
+        char *symbol_name = NULL;
         PyObject *symbol = PyList_GetItem(dir, i);
```

Comment:
(first time I read this code, so apologies if this comment is completely irrelevant)

If [`PyList_GetItem`](https://docs.python.org/2/c-api/list.html#c.PyList_GetItem) returns `NULL`, there is an error set on the interpreter, so we need to handle it, either by cleaning the error (-> discarding it), or returning `NULL` from the function.

Also applies to the `PyString_AsString` usage below, and likely other `Py` functions used in this loop (haven't checked the python C docs on them though).